.github/workflows/dependabot.yaml

```diff
@@ -0,0 +1,97 @@
+name: Pre-merge Checks for Dependabot
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches-ignore: ["version-*"]
```

Comment:
I think dependabot only targets the default branch by default, you can also control in the config file.